examples/chart-move-axis/README.md

```diff
@@ -0,0 +1 @@
+# Chart Move Axis
```

Comment:
Demonstrates how to use decorate to make fundamental changes to how the chart components are laid out.

Reply:
Same goes here, can we add in cartesian?